css-borders-4/Overview.bs

```diff
@@ -617,16 +617,29 @@ Initial: none
 Applies to: all elements
 Inherited: no
 Percentages: N/A
-Computed value: either 'none' or a list,
-	each item a pair of offsets (horizontal and vertical) from the element‘s box
-Animation type: by computed value
+Computed value: list, each item either 'none' or a pair of offsets
+  (horizontal and vertical) from the element‘s box
+Animation type: by computed value,
+    letting ''box-shadow-offset/none'' create a blank shadow
+    (''transparent 0 0 0 0'') when interpolated with non-'none' values
+    and appending blank shadows with a corresponding
+    ''box-shadow-position/inset'' keyword as needed to match the longer list
+    if the shorter list is otherwise compatible with the longer one
 </pre>
 
 <p>The 'box-shadow-offset' property defines one or more drop shadow offsets.
-The property accepts a comma-separated list of horizontal and vertical offset pairs,
+The property accepts a comma-separated list.
+Each item in that list can either be the ''box-shadow-offset/none'' value,
+which indicates no shadow, or a pair of horizontal and vertical offsets,
 where both values are described as <<length>> values.
 
 <dl>
+  <dt><dfn id="shadow-offset-none">none</dfn>
+  <dd>
+    No box shadow will be created.
+    Any other box shadow properties specified for this box shadow have no
+    effect.
```

Comment:
Maybe make it clearer that this only refers to the current shadow? That is, if `none` is mixed with other values, the others can still create shadows.

Reply:
I tried to clarify now that `none` only affects the current shadow.